facter.gemspec

```diff
@@ -24,7 +24,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^bin/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
-  spec.add_development_dependency 'bundler', '~> 2.0'
+  spec.add_development_dependency 'bundler'
```

Comment:
Not sure we would need to have bundler as a dependency at all. Creating a new gem using `bundle gem new_gem` does not add any dependency to bundler.
If we decide to keep it, we should lock it to avoid breaking changes, rubocop is using `'bundler', '>= 1.15.0', '< 3.0'`

Reply:
Removed the bundler dependency.